nifi-nar-bundles/pom.xml

```diff
@@ -93,6 +93,7 @@
         <module>nifi-spark-bundle</module>
         <module>nifi-atlas-bundle</module>
         <module>nifi-druid-bundle</module>
+        <module>nifi-pulsar-bundle</module>
```

Comment:
You need to add a dependency declaration for the NAR in `nifi-assembly/pom.xml`. Just putting that note here since there's no really good place to track that in the review.